workers/unity/Packages/com.improbable.gdk.core/Subscriptions/Systems/WorkerFlagCallbackSystem.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Subscriptions
+{
+    [DisableAutoCreation]
```

Comment:
This needs a group

Reply:
Its never ran

Reply:
Or rather its not ran in the update loop